src/qt/res/src/bitcoin.svg

```diff
@@ -1,58 +1,11 @@
-<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Keep? And license?

Reply:
needs to update https://github.com/bitcoin-core/gui/blob/e130ff38c91ad3527924e6195325f6ae7ce5d77c/contrib/debian/copyright#L81